anitya/templates/project_new.html

```diff
@@ -13,7 +13,7 @@ <h1>{{ context }} project</h1>
 <form method="POST" action="{{ url_for('anitya_ui.new_project') }}" >
 {%- elif context == 'Edit' -%}
 <form method="POST" action="{{ url_for('anitya_ui.edit_project', project_id=project.id) }}" >
-{%- endif %}
+  {%- endif %}
```

Comment:
Don't change indentation of existing files. It will report changes in lines that don't have any changes.